paddle/fluid/framework/feed_fetch_type.h

```diff
@@ -24,6 +25,38 @@ using FeedFetchList = std::vector<FeedFetchType>;
 using FetchUnmergedList = std::vector<std::vector<FeedFetchType>>;
```

Comment:
Here I saw FeedFetchType = LoDTensor, can we using FeedFetchType = boost::variant<LoDTensor, LoDTensorArray>, is the change big?